docs/library/graphing/plotly.md

```diff
@@ -10,88 +10,34 @@ import reflex as rx
 import pandas as pd
 import plotly.express as px
 import plotly.graph_objects as go
-
-def line():
-    df = pd.read_csv("data/canada_life.csv")
-    line = px.line(df, x="year", y="lifeExp", title="Life expectancy in Canada")
-    return line
-
-
-# Read data from a csv
-def mount():
-    z_data = pd.read_csv("data/mt_bruno_elevation.csv")
-    mount = go.Figure(data=[go.Surface(z=z_data.values)])
-    mount.update_traces(
-        contours_z=dict(
-            show=True, usecolormap=True, highlightcolor="limegreen", project_z=True
-        )
-    )
-    mount.update_layout(
-        scene_camera_eye=dict(x=1.87, y=0.88, z=-0.64),
-        width=500,
-        height=500,
-        margin=dict(l=65, r=50, b=65, t=90),
-    )
-    return mount
-
-
-style = {
-    "box": {
-        "borderRadius": "1em",
-        "bg": "white",
-        "boxShadow": "rgba(50, 50, 93, 0.25) 0px 2px 5px -1px, rgba(0, 0, 0, 0.3) 0px 1px 3px -1px",
-        "padding": 5,
-        "width": "100%",
-        "overflow_x": "auto",
-    }
-}
+from pcweb import constants
 ```
 
 Plotly is a graphing library that can be used to create interactive graphs.
```

Comment:
Somewhere in this page we need a clear explanation of how to resize the graph in your app. If it is using `fig.update_layout` or if it is just using the `width` and `height` props.